Cargo.toml

```diff
@@ -1,30 +1,22 @@
 [package]
-name = "tsp"
-version = "0.2.3"
-authors = ["Adrian Brink <adrian@brink-holdings.com>"]
+name = "abci"
+version = "0.3.0"
+authors = ["Adrian Brink <adrian@brink-holdings.com>", "Jackson Lewis <st.japa6@gmail.com>"]
 license = "MIT/Apache-2.0"
-description = "An implementation of the TSP protocol used by Tendermint Core. Allows the building of COSMOS apps in Rust."
+description = "A ABCI server for Rust. Compatible with Tendermint 0.22."
```

Comment:
what about later versions? maybe we should move compatibility matrix to the readme

Reply:
also `An ABCI server ...`

Reply:
What do we put as the description, then?

Reply:
I'll just comment out the description for now.